src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,35 @@
+@import '../utils/mixins';
+
+.stars-area {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  margin-top: 16px;
+
+  &__text {
+    @include product-code-font;
+
+    margin-top: -1px;
+  }
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+
+  &__star {
+    background-image: url(/src/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
```

Comment:
The right margin for the last star should be eliminated to prevent unexpected spacing at the end of the stars block. You can achieve this by using the `:last-child` pseudo-class to target the last star and set its margin-right to 0.